webant/webant.py

```diff
@@ -18,7 +18,52 @@
 from webserver_utils import gevent_run
 
 
-def create_app(configfile=None):
+class LibreantCoreApp(Flask):
+    def __init__(self, import_name):
+        super(LibreantCoreApp, self).__init__(import_name)
+        self.config.update({
+            'BOOTSTRAP_SERVE_LOCAL': True,
+            'DEBUG': True,
+            'PRESET_PATHS': [],  # defaultPreset should be loaded as default?
+            'FSDB_PATH': "",
+            'AGHERANT_DESCRIPTIONS': [],
```

Comment:
ok, corrected. I'm not sure that this is what you mean, but I am sure that this is what I mean :)

Just in case you were thiinking that no default should be in LibreantCoreApp, so the logic can follow one of these options:
1. each parameter HAS to be specified by the initializer. Which is uncommon in python things, whereas sane defaults are much more easy to deal with
2. set default values inside the class, then override them after having it instantiated. This will create "db connections" point to wrong uri (fsdb, elasticsearch, whatever; all would be wrong)

Doing the change you asked, I also moved the option about bootstrap to LibreantViewApp, which is more correct, because LibreantCoreApp does not care about it.